content/1.2/docs/installation/orchestrator-k8s.md

```diff
@@ -42,4 +44,80 @@ kubectl patch daemonsets -n projectcontour envoy -p '{"spec":{"template":{"spec"
 
 The cluster should be up and running with [Contour ingress-controller](https://projectcontour.io) installed, so localhost:9090 will direct the traffic to Backstage, because of the ingress created by the helm chart on port 80.
 
-{{< remoteMD "https://raw.githubusercontent.com/parodos-dev/orchestrator-helm-chart/main/charts/orchestrator-k8s/README.md" >}}
+# Orchestrator-k8s helm chart
+
+This chart will install the Orchestrator and all its dependencies on kubernetes.
+
+**THIS CHART IS NOT SUITED FOR PRODUCTION PURPOSES**, you should only use it for development or tests purposes
+
+The chart deploys:
+
+- RHDH-backstage https://github.com/redhat-developer/rhdh-chart
+- Serverless Workflows Operator (see sonata-serverless-operator.yaml)
+- knative serving
+- Knative eventing
+- Serverless Workflows (optional)
+
+### Usage
+
+```console
+helm repo add orchestrator https://parodos-dev.github.io/orchestrator-helm-chart
+
+helm install orchestrator orchestrator/orchestrator-k8s
+```
+
+## Configuration
+
+All of the backstage app-config is derived from the values.yaml.
+
+### Secrets as env vars:
+
+To use secret as env vars, like the one used for the notification, see charts/Orchestrator-k8s/templates/secret.yaml
+Every key in that secret will be available in the app-config for resolution.
+
+## Development
+
+```console
+git clone https://github.com/parodos-dev.github.io/orchestrator-helm-chart
```

Comment:
the issue with this step - it will always take the latest version.
we don't maintain or enforce any versioning for the k8s chart that can be referred to in respect to Orchestrator for OCP version:
https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator-k8s/Chart.yaml

the options here are to better track and manage versions, or skip that entirely for the k8s chart and use it as a development method for validating the orchestrator (and notifications) plugins by our CI.

if agreed, let not modify this file.